docs/content/Authentication-&-Authorization.md

```diff
@@ -5,23 +5,50 @@ category: Authentication & Authorization
 menuOrder: 1
 ---
 
+In Cube.js, access control or authorization is based on the **security context**. The diagram below shows how it works during the request processing in Cube.js:
+
+
+<p
+  style="text-align: center"
+>
+  <img
+  src="https://raw.githubusercontent.com/statsbotco/cube.js/master/docs/content/authentication-overview.png"
+  style="border: none"
+  width="80%"
+  />
+</p>
+
+Authentication is handled outside of Cube.js. Your auth server issues JWTs to your client app, which, when sent as part of the request, are verified and decoded by the Cube.js to get security context claims to evaluate access control rules.
+
+For access control or authorization, Cube.js allows you to define granular access control rules for evey cube in your data schema. Cube.js uses request and the security context claims in the JWT token to generate a SQL query, which includes row-level constraints from the access control rules.
+
 Cube.js uses [JSON Web Tokens (JWT)][link-jwt] which should be passed in the
-`Authorization` header to authenticate requests. JWTs can also be used for
-passing additional information about the user, which can be accessed in the
-[USER_CONTEXT][link-user-context] object in the Data Schema.
+`Authorization: <JWT>` header to authenticate requests.
 
-[link-jwt]: https://jwt.io/
-[link-user-context]: /cube#context-variables-user-context
+JWTs can also be used to pass additional information about the user, known as a [security context][link-security-context]. It must be stored inside the **u** namespace. It will be accessible in the
```

Comment:
I think that the concept of "namespaces" is quite confusing because we don't explain much about them. It comes literally out of thin air. Can we explain what these namespaces are? Maybe a single sentence will work.